### DISCUSSION TITLE
no login option, only create new account

### DISCUSSION THREAD

**Discussion by paboman** [seriousm4x/UpSnap#218] [Help wanted]
I have installed UpSnap on portainer using this guide: https://mariushosting.com/how-to-install-upsnap-on-your-synology-nas/

Created a new user, then if I logout and want to login again there is only the option to create a new user.
No login option.
what can I check ?
thx

**Comment by paboman** [comment]
I have noticed that even if I type in http://192.168.2.8:8090/login/  I can see the login page for a fraction of second then get redirected on the welcome page. 
same problem on different browser and also different device.

screen recording:
https://www.youtube.com/watch?v=wZQ2Kw4EMwE

**Reply by seriousm4x** [reply]
Strange, the redirect should only happen when the setup hasn't been completed. Have you tried it inside incognito mode?

* Try to clear cookies and website data.
  1. In the URL bar, click on the "not secure" button (if http) / lock icon (if https)
  2. cookies and site data
  3. manage cookies and site data
  4. click the trash can for all items
  5. reload the page

* Check the api response:
  1. Open up developer tools (F12)
  2. Go to network tab and select `Fetch/XHR`
  3. reload the page
  4. Only 1 request should come up, click it and check the `Response` tab on the right. If `"setup_completed"` is false, you will always be redirected to the setup page.
![grafik](https://github.com/seriousm4x/UpSnap/assets/23456686/a5dde14b-5820-47c4-98bf-0700f654e3ae)

**Comment by paboman** [comment]
SOLVED
Rebooting the nas solved the problem
